mangaki/mangaki/utils/mal.py

```diff
@@ -46,8 +47,8 @@ class MALWorks(Enum):
 
 SUPPORTED_MANGAKI_WORKS = [MALWorks.animes, MALWorks.mangas]
 
-
-MALUserWork = namedtuple('MALUserWork', 'title poster mal_id score')
+MALUserWork = namedtuple('MALUserWork', 'title synonyms poster mal_id score')
+MALUserWork.__hash__ = lambda work: hash(work.title)
```

Comment:
I'm fine with inheriting from the `namedtuple`, which still seems better than forcedfully changing the `__hash__`.

```python
class MALUserWork(namedtuple('MALUserWork', ['title', 'synonyms', 'poster', 'mal_id', 'score'])):
    def __hash__(self):
        return hash(self.mal_id)
```